### PR TITLE
Introduce optional support for ansible.cfg

### PR DESCRIPTION
**Pull request by thehedhly**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -97,6 +102,7 @@ jobs:
         with:
           playbook: playbook.yml
           directory: test
+          configuration: ansible.cfg
```
Comment (dawidd6) [inline .github/workflows/test.yml:105]: This looks invalid?

File: test/ansible.cfg [hunk 2]
```diff
@@ -0,0 +1,4 @@
+[defaults]
+callbacks_enabled = ansible.posix.profile_tasks, ansible.posix.timer
+stdout_callback = yaml
+nocows = false
```
[the remark this answers is not in the document]
Comment (thehedhly) [reply test/ansible.cfg:4]: might be needed if overriding the default ansible behaviour is required e.g. using callbacks

File: .github/workflows/test.yml (hunk 1 above)
Comment (thehedhly) [reply .github/workflows/test.yml:105]: I didn't understand what you mean, would you please provide further details

Comment (thehedhly) [reply .github/workflows/test.yml:105]: ah i see local test is failing, am on it

**Comment by dawidd6** [comment]
Thanks, very nice.

**Pull request merged**